go/protocols/capture/capture.proto

```diff
@@ -85,10 +85,13 @@ message Request {
 
   // Discover returns the set of resources available from this connector.
   message Discover {
+    // Name of the capture being discovered.
+    string name = 1
+        [ (gogoproto.casttype) = "github.com/estuary/flow/go/protocols/flow.Capture" ];
     // Connector type addressed by this request.
-    flow.CaptureSpec.ConnectorType connector_type = 1;
+    flow.CaptureSpec.ConnectorType connector_type = 2;
```

Comment:
You cannot change the existing tags of protobuf fields (a central rule of managing protobuf schema evolution), as this severely confuses existing clients / servers and will result in lots of confusing runtime errors.